pytest_plugins/issue_handlers.py

```diff
@@ -197,28 +197,29 @@ def generate_issue_collection(items, config):  # pragma: no cover
 
     # --- Build the issue marked usage collection ---
     for item in items:
-        component = None
-        bz_marks_to_add = []
-        importance = None
+        if item.nodeid.startswith('tests/robottelo/'):
+            # Unit test, no bz processing
+            # TODO: We don't need to be including component and importance in collected_data
```

Comment:
I'm confused. Is this one TODO, or three different TODOs? And if we don't need to include component and importance, why do we?

Reply:
Its one TODO, but couldn't fit on one line. I'll reformat and reword a bit.

Reply:
As far as why do we - couldn't say, but it shouldn't be necessary.  From what I can tell, there is similar behavior here between BZ markers and the testimony tokens, so the functionality was just added into the same function.

I would want to further verify that the issue handlers output data isn't somehow actually using importance or component before removing that data from the dictionary in this function, which is why I'm not including removing it in the PR.